_data/sites.json

```
@@ -6808,6 +6808,19 @@
             "socialblade.com"
         ]
     },
+    
+    {
+        "name": "SoftCreatR Media",
+        "url": "https://www.softcreatr.com/account-management/",
+        "difficulty": "easy",
+        "notes": "Your account will be deleted after 7 days, if you don't cancel the deletion process before. Created content (such as forum posts) won't be deleted, but your username will be anonymized and protected from re-registration for 363 days.",
+        "notes_de": "Dein Benutzerkonto wird nach Ablauf von 7 Tagen gelöscht, solltest du den Vorgang vorher nicht abbrechen. Erstellte Inhalte (wie etwa Foren-Beiträge) werden nicht gelöscht, jedoch wird dein Benutzername anonymisiert und für 363 Tage vor einer erneuten Registrierung geschützt.",
+        "domains": [
+            "www.softcreatr.com",
+            "support.softcreatr.com",
+            "shop.softcreatr.com",
```

```suggestion
            "shop.softcreatr.com"
```
Remove the comma to be a proper JSON